changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: patch
+  changes:
+    added:
+    - Virginia CDCC deduction.
```

(the last C in CDCC is credit)
```suggestion
    - Virginia child and dependent care deduction.
```